disk.go

```diff
@@ -42,8 +87,22 @@ func (d *Disk) scheduleWrite(sizebytes int) time.Duration {
 	at := sizeToDuration(sizebytes, "B", int64(d.MBps), "MB")
 	w := d.writes
 
+	// Simulate adding i/o to the queue.
+	d.qMutex.Lock()
+	d.pendingwbytes += int64(sizebytes)
+	qSz := float64(d.pendingwbytes) / float64(configStorage.sizeDataChunk * 1024)
+	d.qMutex.Unlock()
+
 	d.writes++
 	d.writebytes += int64(sizebytes)
+
+	// Simulate delay based on the queue depth
+	// We do this by a parabola equation of queue depth.
+	// parabola equation :  (x - 32)^2 = 4 * 10 * y
+        latencyF := ((qSz * qSz) - 64.0 * qSz + 1024.0) / 40.0;
+	latency := int64(latencyF * 1000.0)
```

Comment:
Use FIXMEs for temp hardcodings and everywhere else where the stuff is in progress. What's the plan to make this function configurable?